app/(sign-up)/profile-image.js

```diff
@@ -0,0 +1,186 @@
+import {
+  StyleSheet,
+  Image,
+  TouchableOpacity,
+  Modal,
+  ActivityIndicator,
+} from "react-native";
+import React, { useState } from "react";
+import * as ImagePicker from "expo-image-picker";
+import { router } from "expo-router";
+
+//components
+import View from "../../components/ThemedView";
+import Text from "../../components/ThemedText";
+import LoadingAnimation from "../../components/LoadingAnimation";
```

Comment:
then pa check ko doi daw my ara bala sa package.json nga instead mag gamit ka sang ```../../``` amo ni gamiton ```import View from 'component/ThemedView'```

you can also add ``index.js``` sa imo components doi tapos dira mo export tanan para nd kana mag ```../../components/ThemedView``` kay sa iban nga under components lang